psi4/src/psi4/libfock/jk.h

```diff
@@ -753,9 +766,19 @@ class PSI_API DirectJK : public JK {
     /// Post-iteration Incfock processing
     void incfock_postiter();
 
-    /// Build the J and K matrices for this integral class
-    void build_JK(std::vector<std::shared_ptr<TwoBodyAOInt> >& ints, std::vector<std::shared_ptr<Matrix> >& D,
-                  std::vector<std::shared_ptr<Matrix> >& J, std::vector<std::shared_ptr<Matrix> >& K);
+    /// Build the J matrix using the continuous fast multipole method, described in [White:1994:8]_
+    /// TODO: Put CFMM algorithm here after linK PR is merged
+    // void build_cfmm_J(std::vector<std::shared_ptr<TwoBodyAOInt> >& ints, std::vector<SharedMatrix >& D,
+    //               std::vector<SharedMatrix >& J);
+
+    /// Build the K matrix using the linear exchange algorithm, described in [Ochsenfeld:1998:1663]_
+    void build_linK(std::vector<std::shared_ptr<TwoBodyAOInt>>& ints, std::vector<SharedMatrix>& D,
+                  std::vector<SharedMatrix>& J, std::vector<SharedMatrix>& K, bool do_J);
+
+    /// The standard J and K matrix builds for this integral class
+    void build_JK(std::vector<std::shared_ptr<TwoBodyAOInt>>& ints, std::vector<SharedMatrix>& D,
```

Comment:
Mark arguments `const` where possible to signal that this function is supposed to change some arguments, and which argument it's supposed to change.